pipeline/gitlab-ci/dmc.yml

```diff
@@ -0,0 +1,29 @@
+---
+build_dmc:
+  stage: docker_management_container
+  image: docker:git
+  services:
+    - name: docker:dind
+      alias: docker
+  variables:
+    # renovate: datasource=github-releases depName=docker-management-container
+    GITHUB_DMC_TAG: 3.1.0
+    DMC_IMAGE: $CI_REGISTRY_IMAGE/${DMC_NAME}
```

Comment:
`DMC_NAME` is undefined. Let's define a default, like `dmc`, so the include works without any extra steps.